### PR TITLE
Use prettier instead of clang-format 

### PR DESCRIPTION
**Pull request by dayo09**
This commit change the formatter for this project.
Let's use prettier instead of clang-format.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

----

For #1447

### CODE REVIEW

File: infra/format [hunk 1]
```diff
@@ -101,7 +101,7 @@ function check_ts_files() {
   done
 
   if [[ ${#FILES_TO_CHECK_TS} -ne 0 ]]; then
-    ${CLANG_FORMAT} -i ${FILES_TO_CHECK_TS[@]}
+    npx prettier -w ${FILES_TO_CHECK_TS[@]}
```
Comment (dayo09) [inline infra/format:104]: For reviewers, this is the main change (1)

File: package.json [hunk 2]
```diff
@@ -479,6 +479,7 @@
     "mocha": "^9.2.1",
     "mocha-xunit-reporter": "^2.3.0",
     "nyc": "^15.1.0",
+    "prettier": "^2.8.1",
```
Comment (dayo09) [inline package.json:482]: For reviewers, this is the main change (2)

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**